src/rust/ide/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name    = "ide"
+name    = "enso-ide"
```

Comment:
We don't have any other IDEs on this repo that we need to differentiate from and I prefer simpler names.
What's the rationale for this rename?